src/kakarot/account.cairo

```diff
@@ -650,6 +689,26 @@ namespace Account {
         );
         return self;
     }
+
+    func compute_code_hash{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+        code_len: felt, code: felt*
+    ) -> Uint256 {
+        alloc_locals;
```

Comment:
if code_len = 0 you should do an early return with the hash of empty bytecode which is per https://eips.ethereum.org/EIPS/eip-1052 `c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470 `. This might be why you have ef-test failing